front-end/src/components/Spinner.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
```

Comment:
import clsx from 'clsx';

clsx = function to adding the style in the same Component